components/RefinementList.js

```diff
@@ -38,37 +39,68 @@ class RefinementList extends React.Component {
 
   render() {
     var facetValues = this.props.facetValues;
-    var template = this.props.template;
+    var templates = this.props.templates;
+    var rootClass = cx(this.props.cssClasses.root);
+    var listClass = cx(this.props.cssClasses.list);
+    var itemClass = cx(this.props.cssClasses.item);
 
     return (
-      <div className={this.props.rootClass}>
+      <div className={rootClass}>
+        <Template template={templates.header} />
+        <div className={listClass}>
         {facetValues.map(facetValue => {
           return (
-            <div className={this.props.itemClass} key={facetValue.name} onClick={this.handleClick.bind(this, facetValue.name)}>
-              <Template data={facetValue} template={template} />
+            <div className={itemClass} key={facetValue.name} onClick={this.handleClick.bind(this, facetValue.name)}>
+              <Template data={facetValue} template={templates.item} />
             </div>
           );
         })}
+        </div>
+        <Template template={templates.footer} />
       </div>
     );
   }
 }
 
 RefinementList.propTypes = {
-  rootClass: React.PropTypes.oneOfType([
-    React.PropTypes.string,
-    React.PropTypes.arrayOf(React.PropTypes.string)
-  ]),
-  itemClass: React.PropTypes.oneOfType([
-    React.PropTypes.string,
-    React.PropTypes.arrayOf(React.PropTypes.string)
-  ]),
+  cssClasses: React.PropTypes.shape({
```

Comment:
Yes I think we could have them stored somewhere in a util (util.propTypes..). Is that it?